mkdocs.yml

```diff
@@ -111,6 +115,7 @@ plugins:
   - "gen-files":
       scripts:
         - docs/generate_code_reference_pages.py
+  - "glightbox"
```

Comment:
I don't think we want to necessarily auto-lightbox *all* images in the docs. Perhaps https://blueswen.github.io/mkdocs-glightbox/flexibility/enable-by-image-or-page/ ?